lib/cogctl/actions/relays/enable.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Cogctl.Actions.Relays.Enable do
```

Comment:
It would be nice to unify the implementations of the enable and disable actions; it's fine if they have to be separate actions, but they're basically doing the same thing under the hood.